slack_pull_reminder.py

```diff
@@ -61,6 +65,9 @@ def fetch_organization_pulls(organization_name):
     lines = []
 
     for repository in organization.repositories():
+        if REPOS and repository.name.lower() not in REPOS:
+            # if repos list is defined, skip repos not defined in filter
```

Comment:
Think you can remove this comment, the `if` speaks for itself :)

Reply:
done!